### ISSUE TITLE
Adyen

### ISSUE THREAD

**Issue by diegobot1977** [simple-icons/simple-icons#5306]
**Name:** Adyen
**Website:** https://www.adyen.com/
**Alexa rank:** 1,794
**Color:** `#0fbc53`
**Official resources for icon and color:** see below on the Adyen's homepage

**Comment by adamrusted** [comment]
The green looks to be the right choice, and there's an EPS available in their [press kit](https://www.adyen.com/press-and-media) which we can use. Think we can get away with just the `a` as they do in their favicon. Do you want to look at this @diegobot1977 or shall we open it up to others?

**Comment by diegobot1977** [comment]
Yes for sure

**Comment by ericcornelissen** [comment]
Closing this in favor of #5306. @diegobot1977, please open issues for icon requests only once in the future.
